fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -568,6 +568,7 @@ createMaterializedViewStatement
     ('(' columnNameWithComment (',' columnNameWithComment)* ')')?
     comment?
     materializedViewDesc*
+    (TO targetTable=qualifiedName)?
```

Comment:
Just one "TO" I think is not intuitive enough to understand the specific meaning of the grammar

Reply:
Do you have any better idea? this is follow CK's syntax(https://clickhouse.com/docs/en/sql-reference/statements/create/view#materialized-view)